cmd/argo/commands/server.go

```diff
@@ -184,19 +185,60 @@ See %s`, help.ArgoServer),
 	if defaultBaseHRef == "" {
 		defaultBaseHRef = "/"
 	}
+	secure = true
+	if os.Getenv("SECURE") != "" {
+		secure, _ = strconv.ParseBool(os.Getenv("SECURE"))
+	}
+	htst = true
+	if os.Getenv("HTTP_SECURE_TRANSPORT_SECURITY_HEADER") != "" {
+		htst, _ = strconv.ParseBool(os.Getenv("HTTP_SECURE_TRANSPORT_SECURITY_HEADER"))
+	}
+	auth_modes := []string{"client"}
+	if os.Getenv("AUTH_MODES") != "" {
+		auth_modes = strings.Split(os.Getenv("AUTH_MODES"), ",")
+	}
+	configmap := os.Getenv("CONFIGMAP")
+	if configmap == "" {
+		configmap = "workflow-controller-configmap"
+	}
+	namespaced = false
+	if os.Getenv("NAMESPACED") != "" {
+		namespaced, _ = strconv.ParseBool(os.Getenv("NAMESPACED"))
+	}
+	var enable_open_browser bool = false
+	if os.Getenv("BROWSER_ENABLED") != "" {
+		enable_open_browser, _ = strconv.ParseBool(os.Getenv("BROWSER_ENABLED"))
+	}
+	event_operation_queue_size := 16
+	if os.Getenv("EVENT_OPERATION_QUEUE_SIZE") != "" {
+		event_operation_queue_size, _ = strconv.Atoi(os.Getenv("EVENT_OPERATION_QUEUE_SIZE"))
```

Comment:
for int, we might want to error out

Reply:
Try this:

https://github.com/argoproj/argo-workflows/blob/master/util/env/env.go#L24

I think there are also similar ones for booleans.